client/Cargo.toml

```diff
@@ -32,6 +31,7 @@ ureq = { version = "2", default-features = false, features = ["json"] }
 wireguard-control = { path = "../wireguard-control" }
 
 [dev-dependencies]
+lazy_static = "1"
```

Comment:
There's still one last usage of `lazy_static!` in `data_store.rs` tests - is that intentional?

Reply:
Ok, updated :).

Reply:
Hehe, you're quick, thanks. .)